test/cctest/test_util.cc

```diff
@@ -299,3 +303,44 @@ TEST(UtilTest, SPrintF) {
   const std::string with_zero = std::string("a") + '\0' + 'b';
   EXPECT_EQ(SPrintF("%s", with_zero), with_zero);
 }
+
+TEST(UtilTest, PathResolve) {
+#ifdef _WIN32
+  // TODO: figure out how to get env
+  EXPECT_EQ(PathResolve(nullptr, {"c:/blah\\blah", "d:/games", "c:../a"}),
```

Comment:
Wouldn't it be the same as POSIX platforms?

```
  v8::HandleScope handle_scope(isolate_);
  Argv argv;
  Env env{handle_scope, argv};
```

Reply:
Yes, I was waiting to make it work